.teamcity/performance-tests-ci.json

```diff
@@ -1039,4 +1039,4 @@
       }
     } ]
   } ]
-}
+}
```

Comment:
I think we should not remove the end of file newline here. Wonder why `.editorconfig` doesn't ensure that it stays...

Reply:
Jackson's pretty printer doesn't add the newline at the end of the file. Let me see if I can fix this in our code.

Reply:
I added the change so that our generation code adds the newline.